src/widgets/DateRangeSelector.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+import React from 'react';
+import { Text, StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import { DatePickerButton } from './DatePickerButton';
+import { FlexRow } from './FlexRow';
+import { FlexColumn } from './FlexColumn';
+
+import { textStyles, SUSSOL_ORANGE } from '../globalStyles';
+import { generalStrings } from '../localization/index';
+
+export const DateRangeSelector = ({
```

Comment:
tried to make this so that it is able to be used in for example the requisitions page to select a range of entry dates for requisitions to show